docker-app/qfieldcloud/core/templates/admin/project_files_widget.html

```diff
@@ -196,7 +207,8 @@
 
                         $trow.querySelector('td:nth-child(1)').innerHTML = file.name;
                         $trow.querySelector('td:nth-child(2)').innerHTML = file.last_modified;
-                        $trow.querySelector('td:nth-child(3)').innerHTML = file.size;
+                        const [sizeBytes, size10] = filesize10(file.size)
```

Comment:
Avoid returning "tuples" in JS. If you need to return multiple values, better return objects.

In this case, no need to return `sizeBytes` at all, since it is the same as `file.size`.